cmd/internal/flag.go

```diff
@@ -74,7 +74,12 @@ func initMetricsFlags() {
 }
 
 func initKubeconfigFlags(qps float64, burst int) {
-	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
+	kubeconfigExists := flag.Lookup("kubeconfig")
+	if kubeconfigExists == nil {
+		flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
+	} else {
+		kubeconfig = flag.Lookup("kubeconfig").Value.String()
+	}
```

Comment:
Without this change, all the checks fail with of the following error
```
/ko-app/background-controller flag redefined: kubeconfig
panic: /ko-app/background-controller flag redefined: kubeconfig
```

https://github.com/kyverno/kyverno/actions/runs/5660411216/job/15336477938